apps/site/lib/site/application.ex

```diff
@@ -32,10 +32,10 @@ defmodule Site.Application do
         [name: :line_diagram_realtime_cache]
       ]),
       supervisor(Site.GreenLine.Supervisor, []),
-      supervisor(Site.Stream.Vehicles, []),
       supervisor(Site.React, []),
       supervisor(Site.RealtimeSchedule, []),
-      supervisor(SiteWeb.Endpoint, [])
+      supervisor(SiteWeb.Endpoint, []),
+      supervisor(Site.Stream.Vehicles, [])
```

Comment:
😉 Described in e7c769de43a2205d5ce2f0ec7076b664c6fe39ae: 
> `Site.Stream.Vehicles` uses `Endpoint.broadcast` when initializing, but this can fail since `Endpoint` [is not] yet instantiated on `Site.Stream.Vehicles` startup. Fix this by starting the `Endpoint` first.